src/utils/derive_ipfs_cid.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const CID = require('ipfs-only-hash');
```

Comment:
Consider a more generic name for this file like `ipfs_utils.ts`. Naming it for a single function is very limiting.

Reply:
pretty good but the test file name should probably match